tests/Api/Admin/ProductOptionsTest.php

```diff
@@ -95,4 +95,82 @@ public function it_does_not_update_a_product_option_value_with_duplicate_locale_
             Response::HTTP_UNPROCESSABLE_ENTITY,
         );
     }
+
+    /** @test */
+    public function it_updates_a_product_option(): void
+    {
+        $fixtures = $this->loadFixturesFromFiles([
+            'authentication/api_administrator.yaml',
+            'channel.yaml',
+            'product/product_option.yaml',
+        ]);
+        $header = array_merge($this->logInAdminUser('api@example.com'), self::CONTENT_TYPE_HEADER);
+
+        /** @var ProductOptionInterface $productOption */
+        $productOption = $fixtures['product_option_color'];
+
+        $this->client->request(
+            method: 'PUT',
+            uri: sprintf('/api/v2/admin/product-options/%s', $productOption->getCode()),
+            server: $header,
+            content: json_encode([
+                'values' => [
+                    [
+                        'code' => 'CHANGED_COLOR',
+                        'translations' => [
+                            'en_US' => [
+                                'value' => 'Red',
+                            ],
+                        ],
+                    ],
+                ],
+            ], \JSON_THROW_ON_ERROR),
+        );
+
+        $this->assertResponse(
+            $this->client->getResponse(),
+            'admin/product_option/put_product_option',
+            Response::HTTP_OK,
+        );
+    }
+
+    /** @test */
+    public function it_creates_a_product_option(): void
+    {
+        $this->loadFixturesFromFiles([
+            'authentication/api_administrator.yaml',
+            'channel.yaml',
+        ]);
+        $header = array_merge($this->logInAdminUser('api@example.com'), self::CONTENT_TYPE_HEADER);
+
+        $this->client->request(
+            method: 'POST',
+            uri: '/api/v2/admin/product-options',
+            server: $header,
+            content: json_encode([
+                'code' => 'NEW_COLOR',
+                'translations' => [
+                    'en_US' => [
+                        'name' => 'New Color',
+                    ],
+                ],
+                'values' => [
+                    [
+                        'code' => 'NEW_COLOR_RED',
+                        'translations' => [
+                            'en_US' => [
+                                'value' => 'Red',
+                            ],
+                        ],
+                    ],
+                ],
+            ], \JSON_THROW_ON_ERROR),
+        );
+
+        $this->assertResponse(
+            $this->client->getResponse(),
+            'admin/product_option/post_product_option',
```

Comment:
Missing `_response` suffix in the file name, I guess it is not necessary and we can get rid of it, but now, this one file name is not consistent with others that exist.